fastify.js

```diff
@@ -137,6 +141,13 @@ function build (options) {
   fastify._middie = Middie(onRunMiddlewares)
   fastify._middlewares = []
 
+  // schema methods
+  fastify.addSchema = addSchema.bind(ajvContext)
+  fastify.getSchema = getSchema
+  fastify.validate = validate
+  fastify.getStringify = getStringify
+  fastify.stringify = stringify
```

Comment:
can we just expose `addSchema`?

Reply:
Could you clarify what you'd like to see changed? All this is doing is binding the ajv instance that we add a schema to, similar to the default `schemaCompiler`

Reply:
This is adding 5 new methods to fastify. I do not undertand why we need to add validate, getStringify and stringify.